WooCommerce/Classes/Authentication/Prologue/LoginProloguePageViewController.swift

```diff
@@ -49,7 +53,19 @@ final class LoginProloguePageViewController: UIPageViewController {
         pageControl.addTarget(self, action: #selector(handlePageControlValueChanged(sender:)), for: .valueChanged)
     }
 
-    @objc func handlePageControlValueChanged(sender: UIPageControl) {
+    /// Show the next page of content.
+    /// - Returns: Whether there is next page.
```

Comment:
Super nit:


> Returns: Whether there is next page.

To me, this conveys that, I can know if there is a next page after this method returns, which isn't true when I call this method to reach the final page. 
i.e. This method will return `true` when I have already reached the final page.

I hope that this method,
- returns `true` if moving to the next page was successful.
- returns `false` if unable to move to the next page as there is no next page.

Reply:
Thanks for pointing out the confusion in the comment, I updated it in https://github.com/woocommerce/woocommerce-ios/pull/7291/commits/c7dd9ab4d249c7c36cd770a2275570c43fa9f76d - see if it reads better to you 🙂